src/modules/vod_chat/index.js

```diff
@@ -24,6 +91,27 @@ class VODChatModule {
     constructor() {
         watcher.on('vod.message', $el => this.parseMessage($el));
         watcher.on('load.vod', () => $('textarea[data-a-target="video-chat-input"]').attr('maxlength', '500'));
+        watcher.on('load.player', () => this.renderSettings());
+    }
+
+    renderSettings() {
+        if (!$(CHAT_HEADER).length) {
+            setTimeout(() => this.renderSettings(), 100);
+        }
+        $(CHAT_HEADER).append(CHAT_SETTINGS_BUTTON_TEMPLATE);
+        $(CHAT_SETTINGS_WINDOW_TEMPLATE).insertAfter(CHAT_SETTINGS_BUTTON_SELECTOR).hide();
+        chatSettings.load();
+        $(CHAT_SETTINGS_WINDOW_CLOSE_BUTTON_SELECTOR).on('click', () => {
```

Comment:
rather than a close window button, it would be great if clicking out of the popout closes it